airbyte-webapp/src/pages/DestinationPage/pages/DestinationItemPage/components/DestinationSettings.tsx

```diff
@@ -14,6 +14,7 @@ import { ConnectorCard } from "views/Connector/ConnectorCard";
 
 const Content = styled.div`
   max-width: 813px;
+  width: 80%;
```

Comment:
Yes, it will render at 80% of the container until the container is wide enough that 80% > 813px... then it'll be pinned to 813px (not sure why that's the width we use...)